src/exitstrategies/data_elt.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
`exitstrategies` in snake case

Reply:
This is TBD at the next team meet, will be retaining current naming as discussed